.github/workflows/pull-request-test.yml

```diff
@@ -29,4 +29,4 @@ jobs:
           dotnet-version: 6.0.x
 
       - name: Test MonoGame.Extended
-        run: dotnet test --nologo --verbosity minimal --configuration Release
+        run: dotnet test MonoGame.Extended.sln --nologo --verbosity minimal --configuration Release
```

Comment:
Updated the actions to only build MonoGame.Extended.sln

fixes:
`MSBUILD : error MSB1011: Specify which project or solution file to use because this folder contains more than one project or solution file`